Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -2,92 +2,97 @@
   "images" : [
     {
       "idiom" : "iphone",
-      "size" : "20x20",
```

Comment:
There is no real change on this file, just switching order of key/values. Is there any reason to push this?

Reply:
Yeah I realised after pushing, this is something Xcode did. Not sure if they have changed something there

Reply:
If there is no change, can we removed this from the PR please?